src/Twig/Node/GistNode.php

```diff
@@ -31,9 +31,10 @@ public function __construct(
     public function compile(Compiler $compiler): void
     {
         $compiler
+            ->addDebugInfo($this)
             ->write(sprintf(
                 <<<EOT
-echo '<div class="sonata-gist"><script src="https://gist.github.com/%s.js?file=%s"></script></div>';\n,
+echo '<div class="sonata-gist"><script src="https://gist.github.com/%s.js?file=%s"></script></div>';\n
```

Comment:
This was changed from a working code to a non working one here:

https://github.com/sonata-project/SonataFormatterBundle/pull/305

Since 2017 this gist custom tag does not work.